nvflare/private/fed/client/fed_client_base.py

```diff
@@ -27,8 +27,9 @@
 from nvflare.apis.fl_exception import FLCommunicationError
 from nvflare.apis.shareable import Shareable
 from nvflare.apis.signal import Signal
-from nvflare.apis.utils.local_logger import LocalLogger
+from nvflare.app_common.widgets.log_streaming import LocalLogger
```

Comment:
Dependency violation.

Reply:
Thanks, this will be solved by PR #139, I will update this PR after that is merged.